src/Symfony/Component/Messenger/EventListener/SendFailedMessageToFailureTransportListener.php

```diff
@@ -52,11 +52,6 @@ public function onMessageFailed(WorkerMessageFailedEvent $event)
 
         $envelope = $event->getEnvelope();
 
-        // avoid re-sending to the failed sender
-        if (null !== $envelope->last(SentToFailureTransportStamp::class)) {
-            return;
-        }
```

Comment:
If we have the `SentToFailureTransportStamp`, should we avoid adding the stamps below?

Reply:
i thought it was nice semantic to keep for future purposes maybe

Reply:
I can't really think of a downside, other than more and more stamps stacked on each other, but these stamps are tiny...

Reply:
actually it remains used: https://github.com/symfony/symfony/blob/9767270b34e97d1bbf340f9ab3358a0178dccb8a/src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php#L119

Reply:
oh sorry, i misunderstood. We don't need to stack them per se, i agree